lib/generic-number.h

```diff
@@ -104,6 +104,16 @@ gn_is_nan(const GenericNumber *number)
   return _gn_is_nan(number);
 }
 
+gboolean _gn_is_inf(const GenericNumber *number);
+
+static inline gboolean
+gn_is_inf(const GenericNumber *number)
+{
+  if (number->type != GN_DOUBLE)
+    return FALSE;
+  return _gn_is_nan(number);
```

Comment:
Typo

Reply:
The typo has reappeared.

Reply:
good catch. this is an unused function, so I am dropping that patch from the series.

Originally this was used to detect infinite values, but then this would have been difficult due to layering issues.